programme/random-even-number/random-even-number.java

```diff
@@ -0,0 +1,24 @@
+//Program to generate random even number in range using Java
+
+import java.util.Random;
+import java.util.*;
+
+class HelloWorld {
+    public static void main(String[] args) {
+        Scanner s_name = new Scanner(System.in);
+        
+        System.out.println("Enter start range:\n");
+        int startRange = s_name.nextInt();
+        System.out.println("Enter end range:");
+        int endRange = s_name.nextInt();
+        
+        System.out.println("Random value in int from "+startRange+" to "+endRange+ ":\n");
+        int random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
```

Comment:
```suggestion
        System.out.println("Random value in int from " + startRange + " to " + endRange + ":\n");
        int random_int = (int) Math.floor(Math.random() * (endRange - startRange + 1) + startRange);
```

In Java you should add space between operators